### PR TITLE
docs: Add disable_exec_plugin to Machine ID troubleshooting steps

### PR DESCRIPTION
**Pull request by webvictim**
While we wait for https://github.com/gravitational/teleport/issues/40610 to be completed or addressed alternatively, this PR adds a mention of the error shown when `disable_exec_plugin: true` is not set while it should be. This should help with discoverability, as several prospects have hit this issue in the last week or two.

### CODE REVIEW

**Review by strideynet — APPROVED** [review]

File: docs/pages/machine-id/troubleshooting.mdx [hunk 1]
```diff
@@ -292,3 +292,31 @@ flag:
 $ tctl bots rm example
 $ tctl bots add example --roles=foo,bar,machine-id-db
 ```
+
+## Destination kubernetes_secret: `identity-output` must be a directory in exec plugin mode
+
+By default, when outputting a Kubernetes identity, `tbot` outputs make use of a Kubernetes exec
+plugin to always provide the "latest" version of the credentials.
```
Comment (ptgott) [inline docs/pages/machine-id/troubleshooting.mdx:299]: What do the scare quotes mean here? Would the sentence mean the same thing without them?

File: docs/pages/machine-id/troubleshooting.mdx [hunk 2]
```diff
@@ -292,3 +292,31 @@ flag:
...
+    destination:
+      type: kubernetes_secret
+      # For this guide, identity-output is used as the secret name.
+      # You may wish to customize this. Multiple outputs cannot share the same
+      # destination.
+      name: identity-output
+```
+
+Failure to add the `disable_exec_plugin` flag will result in a warning being displayed:
+`Destination kubernetes_secret: identity-output must be a directory in exec plugin mode`
```
Comment (ptgott) [inline docs/pages/machine-id/troubleshooting.mdx:322]:
```suggestion
`Destination kubernetes_secret: identity-output must be a directory in exec plugin mode`.
```
To complete the sentence

**Review by ptgott — APPROVED** [review]

File: docs/pages/machine-id/troubleshooting.mdx (hunk 1 above)
Comment (webvictim) [reply docs/pages/machine-id/troubleshooting.mdx:299]: Yes, good point.

Comment (webvictim) [inline docs/pages/machine-id/troubleshooting.mdx:299]:
```suggestion
plugin to always provide the latest version of the credentials.
```

**Pull request merged**